### DISCUSSION TITLE
PDF Premium

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16587] [Feature improvement]
Kind of not sure so I asking: does Yetiforce PDF Premium is a once time payment (10 Euro) or this is monthly charge?
As this is redirecting to Paypal I assume it is one time payment... How is it?

**Comment by rskrzypczak** [comment]
Yetiforce does not provide add-ons with a one-time fee, unless you want to buy them for a month :).
All add-ons require subscription.
.. so yeah, it's a monthly fee.

**Reply by s2b-git** [reply]
thank you

**Reply by garryatbrighton** [reply]
> Yetiforce does not provide add-ons with a one-time fee, unless you want to buy them for a month :). All add-ons require subscription. .. so yeah, it's a monthly fee.

This is pretty pricey. How about one time fee and a bit more reasonable? Subscription model scares lots of smaller clients and those who's CRM works in intranet only
By the way > █ > This single character adding 400KB to pdf - this is scary too ;-)
